enhancements/api-review/add-ClusterImagePolicy-and-ImagePolicy-for-Signature-Verification.md

```diff
@@ -0,0 +1,634 @@
+
+The merged cluster override policy and namespaced policy in the below `/path/to/policies/testnamespace.json`  will be rolled out. 
+
+```json
+{
+  "default": [
+    {
+      "type": "insecureAcceptAnything"
+    }
+  ],
+  "transports": {
+    "docker": {
+      "test0.com": [
+        {
+          "type": "sigstoreSigned",
+          "fulcio": {
+            "caData": "dGVzdC1jYS1kYXRhLWRhdGE=",
+            "oidcIssuer": "https://OIDC.example.com",
+            "subjectEmail": "test-user@example.com"
+          },
+          "rekorPublicKeyData": "dGVzdC1yZWtvci1rZXktZGF0YQ==",
+          "signedIdentity": {
+            "type": "remapIdentity",
+            "prefix": "test-remap-prefix",
+            "signedPrefix": "test-remap-signed-prefix"
+          }
+        },
+        {
+          "type": "sigstoreSigned",
+          "keyData": "dGVzdC1rZXktZGF0YQ==",
+          "rekorPublicKeyData": "dGVzdC1yZWtvci1rZXktZGF0YQ==",
+          "signedIdentity": {
+            "type": "remapIdentity",
+            "prefix": "test-remap-prefix",
+            "signedPrefix": "test-remap-signed-prefix"
+          }
+        }
+      ],
+      "test1.com": [
+        {
+          "type": "sigstoreSigned",
+          "keyData": "dGVzdC1rZXktZGF0YQ==",
+          "rekorPublicKeyData": "dGVzdC1yZWtvci1rZXktZGF0YQ==",
+          "signedIdentity": {
+            "type": "remapIdentity",
+            "prefix": "test-remap-prefix",
+            "signedPrefix": "test-remap-signed-prefix"
+          }
+        }
+      ],
+      "test2.com": [
+        {
+          "type": "sigstoreSigned",
+          "keyData": "dGVzdC1rZXktZGF0YQ==",
+          "signedIdentity": {
+            "type": "matchRepoDigestOrExact"
+          }
+        }
+      ]
+    },
+    "docker-daemon": {
+      "": [
+        {
+          "type": "insecureAcceptAnything"
+        }
+      ]
+    }
+  }
+}
+```
+
+### Risks and Mitigations
+
+Risk: The ClusterImagePolicy and ImagePolicy policies overwrite
+
+Mitigation: The [Implementation Details](#Update-container-runtime-config-controller-to-watch-ClusterImagePolicy-and-ImagePolicy) merge rule makes sure ImagePolicy for an image cannot override global definitions when merging ClusterImagePolicy and ImagePolicy.
+
+### Drawbacks
+
+## Design Details
+
+### Open Questions [optional]
+
+### Test Plan
+
+**Note:** *Section not required until targeted at a release.*
+
+MCO container runtime config controller can add unit tests and e2e tests.
+- unit test: verify the policies from ClusterImagePolicy and ImagePolicy instances merged correctly and the controller writes correct format policy.json
+- e2e test: 
+  - verify the MCO writes configuration to the correct location. 
+  - test demonstrating that an unsigned image in violation of the policy is rejected
+
+
+### Graduation Criteria
+
+Before GA, we should explore relocating CRD validations to the admission webhook to ensure the scopes are not under OCP image payload repository are configured.
```

Comment:
This isn't quite what I thought we had agreed, I would have said we agreed on
```
Before GA, validation will be implemented to make sure that scopes cannot be committed to an ImagePolicy that are under the scope of a ClusterImagePolicy, this will prevent conflicts between cluster scoped and namespace scoped policy and prevent namespaces attempting to override the global policy.

Additionally, it will be validated that neither an ImagePolicy nor ClusterImagePolicy sets a scope that could conflict with pulling from an OpenShift image payload repository.
```
The important part is that this validation isn't in the CRD, it's additional validation that we cannot currently do within the CRD IIUC